docker-compose.yml

```diff
@@ -104,9 +104,6 @@ services:
       labels:
         polis_tag: ${TAG:-dev}
       args:
-        EMBED_SERVICE_HOSTNAME: ${EMBED_SERVICE_HOSTNAME}
```

Comment:
found the issue @ballPointPenguin - `EMBED_SERVICE_HOSTNAME` accidentally removed - should be good to go now